ibm/resource_ibm_is_image.go

```diff
@@ -379,7 +505,20 @@ func isWaitForImageAvailable(imageC *vpcv1.VpcV1, id string, timeout time.Durati
 
 	return stateConf.WaitForState()
 }
+func isWaitForImageAvailableForVolume(imageC *vpcv1.VpcV1, id string, timeout time.Duration) (interface{}, error) {
+	log.Printf("Waiting for image (%s) to be available.", id)
```

Comment:
remove the log statement if used for debug purpose or else use `fmt` for stdout.